### PR TITLE
Installer Launch sample

### PR DESCRIPTION
**Pull request by dkbennett**
## Description

Adds a sample for the WindowsAppRuntimeInstaller, which is a console application, to be launched by Win32 setup programs without popping up a console window during the setup.

## Checklist

- [x] Samples build and run using the Visual Studio versions listed in the [Windows development docs](https://docs.microsoft.com/windows/apps/windows-app-sdk/set-up-your-development-environment?tabs=stable#2-install-visual-studio).
- [x] Samples build and run on all supported platforms (x64, x86, ARM64) and configurations (Debug, Release). 
- [x] Samples set the minimum supported OS version to Windows 10 version 1809.
- [x] Samples build clean with no warnings or errors.
- [x] **[For new samples]**: Samples have completed the [sample guidelines checklist](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#checklist) and follow [standardization/naming guidelines](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/docs/samples-guidelines.md#standardization-and-naming).

### CODE REVIEW

File: Samples/Installer/cpp-winrt/InstallerLaunchSample/InstallerLaunchSample.sln [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+VisualStudioVersion = 16.0.31702.278
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "InstallerLaunchSample", "InstallerLaunchSample.vcxproj", "{8BF9A634-D043-4D40-AADC-0752DD634E20}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
```
Comment (angelazhangmsft) [inline Samples/Installer/cpp-winrt/InstallerLaunchSample/InstallerLaunchSample.sln:13]: add arm64

File: Samples/Installer/cpp-winrt/InstallerLaunchSample/PropertySheet.props [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (angelazhangmsft) [inline Samples/Installer/cpp-winrt/InstallerLaunchSample/PropertySheet.props:1]: Can probably delete this file

**Review by angelazhangmsft — COMMENTED** [review]
Can we rename the directory to `Samples/Installer/cpp-console-unpackaged`?

And have the sample files directly under that folder to avoid multiple empty directories.

File: Samples/Installer/cpp-winrt/InstallerLaunchSample/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+# Installer launch sample
+
+This sample demonstrates how to launch the WindowsAppRuntimeInstall.exe without having a pop-up
+window appearing while it is running. This sample is intended for Win32 programs which have setup
+programs that need to embed the WindowsAppSDK without any pop-up windows.
+
+## Requirements
+Sample requires downloading or building a version of WindowsAppRuntimeInstall.exe. The sample
+assumes that it will be in %USERPROFILE%\Downloads for convenience. If a different path is
+desired, it can be specified in the #define at the top in main.cpp.
```
Comment (angelazhangmsft) [inline Samples/Installer/cpp-winrt/InstallerLaunchSample/README.md:21]: Maybe add a link to the docs for where you can download WindowsAppRuntimeInstall.exe

File: Samples/Installer/cpp-winrt/InstallerLaunchSample/InstallerLaunchSample.vcxproj [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <PropertyGroup Label="Configuration">
+    <ConfigurationType>Application</ConfigurationType>
+    <PlatformToolset>v143</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)' == '16.0'">v142</PlatformToolset>
```
Comment (erik0686) [inline Samples/Installer/cpp-winrt/InstallerLaunchSample/InstallerLaunchSample.vcxproj:38]: There's a common .props file that samples can use to detect which version of the Platform Toolset should be used :) 
See https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/Samples/Build.Common.Cpp.props
You just need to import it at the top of this file (and potentially add to it the versions for VS15 and VS14. 

I guess another point is, must the samples need to be built with all those versions? It seems the rest of the samples mostly go down to VS16

Comment (angelazhangmsft) [reply Samples/Installer/cpp-winrt/InstallerLaunchSample/InstallerLaunchSample.vcxproj:38]: For WinAppSDK development, the requirement is VS 2019 or higher

File: Samples/Installer/cpp-winrt/InstallerLaunchSample/README.md (hunk 3 above)
Comment (zaryaf) [reply Samples/Installer/cpp-winrt/InstallerLaunchSample/README.md:21]: Can we add a link to the Downloads page to get the installer: https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/downloads

File: Samples/Installer/cpp-winrt/InstallerLaunchSample/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+---
+name: "Installer launch sample" 
+page_type: sample
+languages:
+- cpp
+products:
+- windows
```
Comment (angelazhangmsft) [inline Samples/Installer/cpp-winrt/InstallerLaunchSample/README.md:7]:
```suggestion
- windows-app-sdk
- windows
```

**Review by angelazhangmsft — APPROVED** [review]

**Pull request merged**